modules/javafx.graphics/src/main/native-glass/mac/a11y/AccessibleBase.m

```diff
@@ -97,106 +96,74 @@ - (void)dealloc
     [super dealloc];
 }
 
-- (jobject)getJAccessible
-{
+- (jobject)getJAccessible {
     return self->jAccessible;
 }
 
-- (id)accessibilityValue
+/*
+ * Request accessibility attribute by name from JavaFX Node. Returns attribute value
+ * converted to the native format or NULL if attribute with that name does not exist.
+ * Code that uses this function needs to convert NULL to the default value of a certain type where required.
+ */
+- (id)requestNodeAttribute:(NSString *)attribute
 {
     jobject jresult = NULL;
     GET_MAIN_JENV;
     if (env == NULL) return NULL;
-    jresult = (jobject)(*env)->CallLongMethod(env, self->jAccessible, jAccessibilityAttributeValue, (jlong)@"AXValue");
+    jresult = (jobject)(*env)->CallLongMethod(env, [self getJAccessible],
```

Comment:
not an expert, but do we really need to initialize `jresult` to NULL in line 110?
can the variable be declared in line 113 ?

Reply:
Well this pattern was used to initialize the result value to the default return value so if assignment does not happen due to the Java exception we have something meaningful to return but in case of NULL it is not really required because it will produce the same result anyways.